.gitignore

```diff
@@ -9,4 +9,5 @@ jquery.pubsub.js
 .nyc_output/
 coverage
 # yarn lock file
-yarn.lock
\ No newline at end of file
+yarn.lock
+.vscode
```

Comment:
The local .gitignore is for files produces by code in the repository.
Neither of these are that.

I'd recommend putting the `.vscode` entry in your [global `.gitignore`](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html) and deleting the other one.